src/simplicial-hott/09-yoneda.rzk.md

```diff
@@ -152,16 +157,15 @@ in two steps.
       ( \ f -> yon-evid-twice-pointwise ϕ x f)
 
 -- By funext again, these are equal as functions of x and f.
-#def yon-evid
-  (funext : FunExt)
+#def yon-evid uses (funext)
   (ϕ : (z : A) -> hom A a z -> C z)
   : ((yon A is-segal-A a C is-covariant-C) ((evid A a C) ϕ )) = ϕ
   := eq-htpy funext
```

Comment:
You may notice that some occurrences of `funext` (and `extext`) in arguments is not removed. This is because definitions that use them come from different files, so the argument becomes explicit.

I think it makes sense have a (global) version of `#assume` that will maintain assumption implicit, even in other files. I will try to introduce such a feature in future versions of `rzk`.